schedule/yast/ssh-x.yaml

```diff
@@ -0,0 +1,42 @@
+name:           ssh-X
+description:    >
+  Conduct an installation using ssh with X-Forwarding.
+  Might only be effective for zVM and powerVM
+vars:
+  DESKTOP: textmode
+  PATTERNS: minimal,base
+  VIDEOMODE: ssh-x
+schedule:
+  - installation/bootloader_start
+  - installation/welcome
+  - installation/accept_license
+  # Required on zVM
+  - '{{disk_activation}}'
+  - installation/scc_registration
+  - installation/addon_products_sle
+  - installation/system_role
+  - installation/partitioning
+  - installation/partitioning_filesystem
+  - installation/partitioning_finish
+  - installation/installer_timezone
+  - installation/user_settings
+  - installation/user_settings_root
```

Comment:
I cannot see 'resolve_dependency_issues' or 'select_patterns', maybe there is reason to not include them?

Reply:
I've unscheduled `resolve_dependency_issues` on purpose, as it would hide dependency issues, but thanks for mentioning `select_patterns`, because I missed pushing it. Cheers!